### PR TITLE
[plugin.video.rtbfauvio] 0.0.1

### PR DESCRIPTION
**Pull request by Gaet81**
Add to Official Repository

### CODE REVIEW

File: plugin.video.rtbfauvio/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<addon id="plugin.video.rtbf.auvio"
+       name="RTBF Auvio"
+       version="0.0.1"
+       provider-name="Gaet">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.elementtree" version="1.2.7"/>
+    <import addon="script.module.simplejson" version="2.0.10"/>
+    <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+    <import addon="script.module.m3u8" version="0.2.2"/>
```
Comment (phil65) [inline plugin.video.rtbfauvio/addon.xml]: you dont use most of these so please remove them from this list.

File: plugin.video.rtbfauvio/addon.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rtbf.auvio"
+       name="RTBF Auvio"
+       version="0.0.1"
+       provider-name="Gaet">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```
Comment (phil65) [inline plugin.video.rtbfauvio/addon.xml]: this must be 2.24.0

File: plugin.video.rtbfauvio/addon.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  </requires>
+  <extension point="xbmc.python.pluginsource"
+            library="addon.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary>replay TV and Radio Shows from French Belgian Public broadcaster</summary>
+    <description>Find all replay TV/Radio shows (mainly limited to Belgium due to geo limitation)</description>
+    <platform>all</platform>
+    <license>WTFPL</license>
```
Comment (phil65) [inline plugin.video.rtbfauvio/addon.xml]: add a proper license file, please

Comment (kib) [inline plugin.video.rtbfauvio/addon.xml]: You want him to add the WTFPL ? http://www.wtfpl.net/txt/copying/

Comment (MartijnKaijser) [inline plugin.video.rtbfauvio/addon.xml]: I don't see why not.

**Comment by Gaet81** [comment]
Hello, sorry will close this pull request because it will be included in another plugin

**Pull request closed without merging**